tensorflow_recommenders_addons/dynamic_embedding/python/keras/layers/dynamic_layer_normalization.py

```diff
@@ -0,0 +1,96 @@
+import tensorflow as tf
+from tensorflow.keras.layers import LayerNormalization
+
+
+class DynamicLayerNormalization(LayerNormalization):
```

Comment:
It is recommended to keep the name of `LayerNormalization` and just use a different prefix sort of `de.keras.layers.LayerNormalization`. Accordingly, the import statement needs to be modified to `from tensorflow.keras.layers import TFLayerNormalization`

Reply:
you mean from tensorflow.keras.layers import LayerNormalization as TFLayerNormalization ?

Reply:
Correct! And using `de.keras.layers.LayerNormalization`

Reply:
how about de.LayerNormalization?